api/src/main/java/com/nicknathanjustin/streamercontracts/users/UserModel.java

```diff
@@ -0,0 +1,30 @@
+package com.nicknathanjustin.streamercontracts.users;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "users")
+@Builder
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@Setter(value = AccessLevel.PACKAGE)
+@Getter
+public class UserModel {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
```

Comment:
Why not a UUID? Also, does this generate sequential ids? Sequential keys are typically considered a no-no from what ive seen

Reply:
As discussed, changing this to UUIDs.